src/app/src/components/ClaimFacilityStepper.jsx

```diff
@@ -0,0 +1,226 @@
+import React, { useState, useEffect } from 'react';
+import { bool, func, string } from 'prop-types';
+import { connect } from 'react-redux';
+import Stepper from '@material-ui/core/Stepper';
+import Step from '@material-ui/core/Step';
+import StepLabel from '@material-ui/core/StepLabel';
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+import clamp from 'lodash/clamp';
+import last from 'lodash/last';
+import { isEmail, isEmpty } from 'validator';
+
+import BadgeClaimed from './BadgeClaimed';
+import ClaimFacilityContactInfoStep from './ClaimFacilityContactInfoStep';
+import ClaimFacilityFacilityInfoStep from './ClaimFacilityFacilityInfoStep';
+import ClaimFacilityFinishUpStep from './ClaimFacilityFinishUpStep';
+
+import {
+    submitClaimAFacilityData,
+    updateClaimAFacilityEmail,
+} from '../actions/claimFacility';
+
+const claimFacilityStepperStyles = Object.freeze({
+    containerStyles: Object.freeze({
+        padding: '10px 20px 5px',
+    }),
+    buttonsContainerStyles: Object.freeze({
+        padding: '24px 0',
+    }),
+    buttonStyles: Object.freeze({
+        margin: '10px',
+    }),
+    formContainerStyles: Object.freeze({
+        width: '100%',
+        padding: '20px',
+        display: 'flex',
+        flexDirection: 'column',
+    }),
+    validationMessageStyles: Object.freeze({
+        padding: '5px 0',
+    }),
+});
+
+const steps = Object.freeze([
+    Object.freeze({
+        name: 'Contact Information',
+        component: ClaimFacilityContactInfoStep,
+        next: 'Facility Information',
+    }),
+    Object.freeze({
+        name: 'Facility Information',
+        component: ClaimFacilityFacilityInfoStep,
+        next: 'Finish Up',
+    }),
+    Object.freeze({
+        name: 'Finish Up',
+        component: ClaimFacilityFinishUpStep,
+        next: 'Thank you',
+    }),
+]);
+
+function ClaimFacilityStepper({
+    fetching,
+    submitClaimForm,
+    email,
+    companyName,
+    contributorEmail,
+    setEmailFromContributorEmail,
+}) {
+    const [activeStep, setActiveStep] = useState(0);
+
+    useEffect(() => {
+        setEmailFromContributorEmail(contributorEmail);
+    }, [contributorEmail, setEmailFromContributorEmail]);
```

Comment:
This will only fire on mount and when `contributorEmail` and `setEmailFromContributorEmail` props change -- which will not happen while the component is mounted, so it will only fire once.

There's a hooks linting rule to require that one declare all dependencies of `useEffect` in the array, which is why I've added them here.